### ISSUE TITLE
get_fresh breaks configuration

### ISSUE THREAD

**Issue by shughes-uk** [rochacbruno/dynaconf#129]
It's possible i'm using this wrong but I suspect this might be bug.

```Dynaconf 1.2.1,``` 
```
Python 3.7.1 (default, Dec 14 2018, 13:28:58) 
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
```
Code
```
In [1]: from dynaconf import settings                                                                                                                                                    

In [2]: settings.POTATO                                                                                                                                                                  
Out[2]: 'test'

In [3]: settings.get_fresh('POTATO')                                                                                                                                                     

In [4]: settings.POTATO                                                                                                                                                                  
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-22d467a777a4> in <module>
----> 1 settings.POTATO

~/miniconda3/lib/python3.7/site-packages/dynaconf/base.py in __getattr__(self, name)
     95             raise AttributeError(
     96                 "Attribute %s was deleted, "
---> 97                 "or belongs to different env" % name
     98             )
     99         if (

AttributeError: Attribute POTATO was deleted, or belongs to different env
```
settings.toml
```
[default]# Defaults
   potato='test'
```

empty .env file.

**Comment by rochacbruno** [comment]
It looks like a bug, wilk try to reproduce. Thanks for reporting.

**Comment by rochacbruno** [comment]
The fix is coming on #130 

(next release will include more bug fixes over this week)
